bit_manipulation/power_of_2.cpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file
+ * @brief [Find whether a given number is power of 2]
+ * (https://www.geeksforgeeks.org/program-to-find-whether-a-given-number-is-power-of-2/) implementation
+ *
+ * @details
+ * We are given a positive integer number. We need to check whether the number is power of
+ * 2 or not.
+ *
+ * A binary number consists of two digits. They are 0 & 1. Digit 1 is known as
+ * set bit in computer terms.
+ * Worst Case Time Complexity: O(1)
+ * Space complexity: O(1)
+ * @author [Prafful Gupta](https://github.com/EcstaticPG-25811)
+ */
+
+#include <cassert>   /// for assert
+#include <iostream>  /// for IO operations
+
+/**
+ * @namespace bit_manipulation
+ * @brief Bit manipulation algorithms
+ */
+namespace bit_manipulation {
```

Comment:
Why 2 namespaces are required ? Is it like you would add more algorithms in this file ?

Reply:
Actually, only two namespaces are required in case there are a lot of functions. But in this case, it's only one function, so we can use `bit_manipulation` only.